ansible/roles/nim-waku/templates/docker-compose.yml.j2

```diff
@@ -42,6 +42,7 @@ services:
       --keep-alive=true
       --persist-messages=true
       --lightpush=true
+      --swap=true
```

Comment:
Wasn't this supposed to be parametrized based on fleet(`test` or `prod`).

Reply:
Yea, made the change. I enabled swap in the test fleet for now but not the prod fleet. 
But Does this mean that the change in the docker-compose.yml is not necessary?

Reply:
Okay, but you're still missing a conditional in the `docker-compose.yml.j2` template that would make the `--swap=true` actually dependent on the value of `nim_waku_swap_protocol_enabled`. Now, there are two way sto do this:

#### Jinja2 Conditional
```j2
{% if nim_waku_swap_protocol_enabled %}
      --swap=true
{% endif %}
```
This would make things less verbose when it's disabled, but considering we can provide a value there's a simpler way:

#### Templated Value
```j2
      --swap={{ nim_waku_swap_protocol_enabled | lower }}
```
Here we add the `lower` filter to make sure the boolean value gets converted to a lower case string.

I'd say the second way makes for a cleaner template.